### DISCUSSION TITLE
Issues adding data packs

### DISCUSSION THREAD

**Discussion by Mika3002** [jacobsjo/mc-datapack-map#8] [Q&A] [answered]
Hello, I'm trying to add the continents mod for testing, but it seems to not be loading. I may be doing something wrong, so any help is appreciated

**Comment by jacobsjo** [comment, marked as answer]
I assume you are referring to the continents datapack from Stardust Labs, correct? All you have to do is to download the continents datapack and then open the zip file in the map:

![image](https://github.com/jacobsjo/mc-datapack-map/assets/10163794/5d827094-8b18-4ce3-928e-048fd4719fe5)

If you have any more problems with that, let me know!
